euphonic/readers/castep.py

```diff
@@ -193,12 +196,13 @@ def read_phonon_data(
         loto_split_indices = set()
 
         while True:
```

Comment:
And here

Reply:
The Walrus Will Return

Reply:
`while (frequency_block := _read_requency_block(...))` would be lovely :seal:

Reply:
No need for the parens even.

Reply:
Ah, didn't realise it has super-low priority but that makes sense :smile: